docs/source/getting-started/triggers.rst

```diff
@@ -0,0 +1,265 @@
+.. Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+A Simple Chatbot
+################
+
+So far, the application is entirely driven by users interacting with the
+browser. Sometimes, your application design will require that some actions are
+taken automatically, or some users will want to automate their own behaviour on
+the ledger. This can be done through various APIs and using various languages,
+but in this section we'll give you a quick tour of the simplest route to
+automation: :doc:`Daml Triggers </triggers/index>`.
+
+Objective
+*********
+
+We're going to build a chatbot that answers every message with
+
+  "Please, tell me more about that."
+
+That should fool anyone and pass the Turing test, easily.
+
+A No-Op Trigger
```

Comment:
At the risk of sidetracking this I wonder if an autofollow trigger that follows everyone that follows you might be a better example:
1. It doesn’t require any changes to the model.
2. It is easy to make it robust.
3. The code itself should be simpler as well.